js/views/PageNav.js

```diff
@@ -0,0 +1,67 @@
+import electron from 'electron';
+import { View } from 'backbone';
+import loadTemplate from '../utils/loadTemplate';
+import app from '../app';
+
+const remote = electron.remote;
+
+export default class PageNav extends View {
```

Comment:
Yes, the pageNav / address bar should always be available. Horrible UX to block it.